extensions/standard-processors/tests/features/file_system_operations.feature

```diff
@@ -19,49 +19,57 @@ Feature: File system operations are handled by the GetFile, PutFile, ListFile an
   As a user of MiNiFi
   I need to have GetFile, PutFile, ListFile and FetchFile processors
 
-  Background:
-    Given the content of "/tmp/output" is monitored
-
   Scenario: Get and put operations run in a simple flow
     Given a GetFile processor with the "Input Directory" property set to "/tmp/input"
     And a file with the content "test" is present in "/tmp/input"
     And a PutFile processor with the "Directory" property set to "/tmp/output"
+    And PutFile is EVENT_DRIVEN
     And the "success" relationship of the GetFile processor is connected to the PutFile
+    And PutFile's success relationship is auto-terminated
     When the MiNiFi instance starts up
-    Then a flowfile with the content "test" is placed in the monitored directory in less than 10 seconds
+    Then there is a single file with "test" content in the "/tmp/output" directory in less than 10 seconds
 
   Scenario: PutFile does not overwrite a file that already exists
-    Given a set of processors:
-      | type    | name      | uuid                                 |
-      | GetFile | GetFile   | 66259995-11da-41df-bff7-e262d5f6d7c9 |
-      | PutFile | PutFile_1 | 694423a0-26f3-4e95-9f9f-c03b6d6c189d |
-      | PutFile | PutFile_2 | f37e51e9-ad67-4e16-9dc6-ad853b0933e3 |
-      | PutFile | PutFile_3 | f37e51e9-ad67-4e16-9dc6-ad853b0933e4 |
+    Given a GetFile processor with the "Input Directory" property set to "/tmp/input"
+    And a PutFile processor with the name "PutFile_1"
+    And PutFile_1 is EVENT_DRIVEN
+    And a PutFile processor with the name "PutFile_2"
+    And PutFile_2 is EVENT_DRIVEN
+    And a PutFile processor with the name "PutFile_3"
+    And PutFile_3 is EVENT_DRIVEN
 
-    And these processor properties are set:
+    And these processor properties are set
       | processor name | property name   | property value |
       | GetFile        | Input Directory | /tmp/input     |
-      | PutFile_1      | Input Directory | /tmp           |
+      | PutFile_1      | Directory       | /tmp           |
```

Comment:
interesting -- how was this working before? PutFile has no Input Directory property